### ISSUE TITLE
Bugs while releasing to Github Releases

### ISSUE THREAD

**Issue by amir20** [release-it/release-it#763]
I use `release-it` to automate my release process. Tags are done locally and pushed to Github. Once Github Actions does the release, it will create a release page. 

Here are [relevant](https://github.com/amir20/dozzle/blob/master/.github/workflows/deploy.yml#L100) scripts I use. It does `yarn release --github.release --no-increment --no-git --ci` which only releases. 

## Actual Behavior

However, there are two things wrong and I can't make sense. 

1. The release date is March 30th. However, I just did the release today so that is weird. See https://github.com/amir20/dozzle/releases/tag/v3.6.5
2. It seems like all previous releases are dropped and only the latest one captures a changelog. See https://github.com/amir20/dozzle/releases

## Expected Behavior

I would expect the date to be today. Also, changelog should not be changed for previous releases. 

Am I doing something that is causing this to happen. Thanks.

**Comment by webpro** [comment]
When using the `--no-increment` flag, the latest GitHub Release is updated.

To get the latest release, release-it uses the [Octokit API listReleases endpoint](https://octokit.github.io/rest.js/v18#repos-list-releases). However, from their docs:

> This returns a list of releases, which does not include regular Git tags that have not been associated with a release. To get a list of Git tags, use the Repository Tags API.

So I guess that's the issue: release-it does not _create_ a GitHub Release for the latest _unreleased_ Git tag, but _update_ the latest GitHub _Release_.

Looking at [the release-it docs](https://github.com/release-it/release-it/blob/master/docs/github-releases.md#update-the-latest-release), this behavior is not wrong, but I think release-it's behavior should be changed to "create or update the GitHub Release associated with the latest tag". That makes sense and when using `--no-increment` to re-use the latest version/tag.

Not sure if this is a bugfix or a breaking change; I tend to the former.

Last but not least, I'm not 100% sure this change will fix your issue, but it's likely.

**Comment by webpro** [comment]
A bugfix is in [v14.6.2](https://github.com/release-it/release-it/releases/tag/14.6.2), hope this helps!

**Comment by amir20** [comment]
Thanks @webpro   I’ll give it a try this week. 

Any idea why the dates are wrong too?

**Comment by webpro** [comment]
No, not really. In the end, release-it only creates or updates a single GitHub Release using the Octokit REST API. There is nothing regarding the concept of a release timestamp.

**Comment by amir20** [comment]
Thanks @webpro. Looks like it is fixed now!

https://github.com/amir20/dozzle/releases

And dates have been fixed too. I guess it was related.

**Comment by amir20** [comment]
Closing.

**Comment by webpro** [comment]
That's great, thanks for the feedback!
